beacon-chain/blockchain/core.go

```diff
@@ -195,7 +195,7 @@ func (b *BeaconChain) CanProcessBlock(fetcher types.POWBlockFetcher, block *type
 	if err != nil {
 		return false, err
 	}
-	if !hasParent {
+	if !hasParent && block.SlotNumber() != 1 {
 		return false, errors.New("parent hash points to nil in beaconDB")
```

Comment:
Can you clarify this? I found it confusing that it was an ERROR level logging and what caused it.

Reply:
These should be debug level, basically saying that the new incoming block either had (1) wrong state hashes, (2) has no parent in the db, or (3) has no valid mainchain reference